src/components/CaseEditor/CaseEditor.js

```diff
@@ -70,13 +71,24 @@ class _CaseEditor extends Component {
     this.handleNewImg = this.handleNewImg.bind(this);
     this.deleteImg = this.deleteImg.bind(this);
     this.deleteLead = this.deleteLead.bind(this);
+    this.handleScroll = this.handleScroll.bind(this);
     this.state = {
       lead: "",
       newImg: false,
-      delImg: false
+      delImg: false,
+      fixedBar: false,
     };
   }
 
+  handleScroll() {
+    let height = window.scrollY;
+    if (height > 540) {
```

Comment:
Y position of the toolbar will undoubtedly vary depending on whether there are images or not, zoom factor, etc. no?

I haven't studied the code, but if you've got a handler on scroll, can't we just detect being offscreen? e.g. http://stackoverflow.com/questions/123999/how-to-tell-if-a-dom-element-is-visible-in-the-current-viewport/7557433#7557433